tasks/models.py

```diff
@@ -1,3 +1,39 @@
+from django.contrib.auth.models import User
 from django.db import models
 
-# Create your models here.
+
+class Tag(models.Model):
+    tag_name = models.CharField(max_length=50, unique=True, primary_key=True)
+
+    def __str__(self):
+        return self.tag_name
+
+
+class Task(models.Model):
+    task_code = models.CharField(max_length=8, unique=True, primary_key=True)
```

Comment:
max_length should be more than that, maybe 16 (in my opinion)

Reply:
Also, task_code should just be a unique column, and other tables will reference the id column of Task. Since task_code might be modified, if task_code is the primary key then every references to the old task_code must be updated. There should be no performance impact (or if there is, very small impact) between querying a primary key column vs. querying a unique key column.